cdc/sink/codec/canal_flat.go

```diff
@@ -39,6 +42,36 @@ type CanalFlatEventBatchEncoder struct {
 
 const tidbWaterMarkType = "TIDB_WATERMARK"
 
+var str2MySQLType = map[string]byte{
```

Comment:
At the moment, it's a reverse map of the `https://github.com/pingcap/tidb/blob/master/parser/types/etc.go#L43`.

But it would be hard to track TiDB type changes. If we would update the TiDB dependency in the go mod file, we have to manually check for this.

Reply:
How about exporting `type2Str` in parser and reversing map at runtime?

Reply:
If we can find a way to only reversing the map once, I think exporting `type2Str` is a good choice, otherwise, I would prefer move `str2MySQLType` to the parser directly.